src/utils/commentedOutCode.js

```diff
@@ -0,0 +1,56 @@
+// const currentUser = getUser();
```

Comment:
Probably want to avoid storing reference code in the repo. Removing old/unused/commented-out code before PR is ideal.